packages/rrweb-snapshot/src/utils.ts

```diff
@@ -167,6 +167,12 @@ export function maskInputValue({
   maskInputFn?: MaskInputFn;
 }): string {
   let text = value || '';
+
+  // Handle `option` like `select
+  if (tagName.toLowerCase() === 'option') {
+    tagName = 'select';
+  }
+
```

Comment:
This method always gets an upper case tagName, keeping it upper case would save us one `toLowerCase` call, slightly improving performance
```suggestion
  // Handle `option` like `select
  if (tagName === 'OPTION') {
    tagName = 'SELECT';
  }

```

Reply:
Yeah, I wasn't quite sure here myself, as sometimes tagNames are passed as lowercase, sometimes as uppercase strings 😅

Reply:
Oops, actually it is _not_ always lowercase (which nicely the tests caught!). 
I've actually changed the type to use `Uppercase` to ensure that the correct case is passed in. This is supported since TS 4.1, is this OK?